install.sh

```diff
@@ -167,6 +171,60 @@ legacy_transformations_start() {
     return 0
 }
 
+############################################################
+# Tests if the commit before a feature is introduced `$2`
+#   is after or equal the commit before the update `$1`.
+#   Note: Any feature that this install.sh knows about
+#   is an ancestor of the current version.
+#
+# Returns: 0 if succesful, 1 otherwise
+############################################################
+legacy_transformation_update_includes() {
```

Comment:
Could we improve this function name? To me, it keeps registering as if this function is going to update the includes. As I suggested before, I wouldn't mind making the first argument explicit, in which case `legacy_transform_updating_from_before [hash]` might be more readable? If we leave the second argument, how about just say what it does: `legacy_transform_is_ancestor`?

Reply:
Jup, nice!